modules/ducktests/tests/ignitetest/tests/auth_test.py

```diff
@@ -24,12 +24,13 @@
 from ignitetest.services.utils.ignite_configuration import IgniteConfiguration, DataStorageConfiguration
 from ignitetest.services.utils.ignite_configuration.data_storage import DataRegionConfiguration
 from ignitetest.utils import ignite_versions, cluster
-from ignitetest.services.utils.ignite_configuration.discovery import from_ignite_cluster
+from ignitetest.services.utils.ignite_configuration import IgniteThinClientConfiguration
+from ignitetest.services.utils.ssl.client_connector_configuration import ClientConnectorConfiguration
 from ignitetest.utils.ignite_test import IgniteTest
 from ignitetest.utils.version import DEV_BRANCH, LATEST, IgniteVersion
 
 WRONG_PASSWORD = "wrong_password"
-TEST_USERNAME = "admin"
+TEST_USERNAME = "ADMIN"
```

Comment:
why uppercase now?

Reply:
There are 2 ways to pass username in SQL query:
1. case-sensitive
    "CREATE USER \"%s\" WITH PASSWORD '%s';"
2. not case-sensitive
    "CREATE USER %s WITH PASSWORD '%s';"
 IMHO variant 2 looks cleaner (but username should be uppercase)

Reply:
`TEST_USERNAME` implicitly connected with `UserModifyingApplication` - username should be in uppercase to work correctly.

It seems wrong connection. 
Please, fix the app and add quotes to queries to make queries not related to parameter case.